frontend/src/app/features/details/SummaryForm.tsx

```diff
@@ -23,7 +26,25 @@ const SummaryForm: FC<SummaryFormProps> = ({
   srMode,
   changeHandler,
 }) => {
-  const formRows: IFormField[][] = [
+
+  const userPurchasedSnapshot = useSelector(isUserPurchasedSnapshot);
+
+  const [finalFormConfiguration, SetFinalFormConfiguration] = useState<IFormField[][]>([]); 
```

Comment:
pls follow camelCase convention for `SetFinalFormConfiguration`

Reply:
done